src/pages/installing-spark/react.mdx

```diff
@@ -2,142 +2,107 @@
+If your React server has stopped, restart it now and view your compiled React site. You should see the new CSS you’ve written included in the site, along with the original React CSS:
 
-IMAGE
 
-## 🧱 Installing Spark Packages
+<div class="sprk-u-Measure">
+  <img
+    class="sprk-u-mbm "
+    style="margin: auto; display: block;"
+    src={reactApp}
+    alt="Screenshot of Create React App running in a browser."/>
+</div>
 
-All the HTML, JavaScript, and CSS needed to run Spark React components is available in our npm packages. You can download the required packages with this command:
+## Install Spark Packages
+Now we’ll use Sass to import all the Spark CSS.
 
+6. Download the required packages with this command. It contains all of the HTML, JavaScript, and CSS needed to run Spark React components.
 ```
 npm install @sparkdesignsystem/spark @sparkdesignsystem/spark-react --save-dev
 ```
-
-Now that you have the files installed, you need to import the Spark Sass file into the main Sass file for the project (e.g., App.scss).
-
+7. Import the Spark Sass file into the main Sass file (e.g., `App.scss`).
 ```
 @import '../node_modules/@sparkdesignsystem/spark/spark.scss';
 ```
+When you build and run your website, all the Spark CSS will now be bundled into your output. You can verify this by inspecting your site’s DOM and looking for the Spark CSS. There should be a `<style>` tag in the `<head>` of your page that includes Spark’s styles.
+<div class="sprk-u-Measure">
+  <img
+    class="sprk-u-mbm "
+    style="margin: auto; display: block;"
+    src={styleCheck}
+    alt="Screenshot of DOM with Spark Styles"/>
+</div>
+If Spark React is installed correctly, you should see these CSS rules listed in your page’s code.
 
-Now when you build and run your website, all the Spark CSS will be bundled into your output. The easiest way to verify that this is working is to add a Spark component, which we will do in the next step. You can also verify that the Spark CSS is being included by inspecting your site’s DOM and looking for the Spark CSS. There should be a `<style>` tag in the `<head>` of your page that includes all the Spark styles.
-
-IMAGE
-
-## 🧭 Navigating to Component Code Samples
-
-On sparkdesignsystem.com, we’ve documented all the Spark components with copy/paste examples for many variants and configurations you can use in your site.
-
-Let’s include a Spark Button component from the site.
-
-TODO: Instructions on accessing the Gatsby/Storybook site and getting to the React copy/paste code. Fill this section in once the site navigation is working.
+## Adding React Components
+Let’s add a component that will allow us to test that both Spark’s styles and functionality are working. We’ll use the [Loading Button](https://angular.sparkdesignsystem.com/?path=/docs/components-button--loading) component to do this. You can find code for all Spark components at [react.sparkdesignsystem.com](https://react.sparkdesignsystem.com/).
```

Comment:
This link goes to the angular version and not the react version. 

[Loading Button](https://angular.sparkdesignsystem.com/?path=/docs/components-button--loading)